webapp/src/routes/EditProfile/EditProfileBankMobile.js

```diff
@@ -522,7 +522,6 @@ const EditProfileBankMobile = ({ profile, onSubmit, setField, loading, userName
               InputLabelProps={{
                 shrink: true
               }}
-              className={classes.textField}
```

Comment:
@Edgar097 could you please check if this no cause any problem, the thing is that there was two `className `declaration to same component.